<launch>

    <param name="scan_skip" type="int" value="0" />
    <param name="sector_num" type="int" value="8" />
    <param name="sharp_num" type="int" value="2" />
    <param name="sharpless_num" type="int" value="10" />
    <param name="flat_num" type="int" value="2" />
    <param name="flatless_ds" type="double" value="0.2" />
    <param name="skip_frame" type="int" value="1" />
    <param name="min_range" type="double" value="0.5" />
    <param name="max_range" type="double" value="20.0" />

    
    <!-- Change the topic name if use and recorded .bag -->
    <param name="topic_name" type="string" value="horLidar/hor_lidar_points_cloud" />
    <!-- <param name="topic_name" type="string" value="/lslidar_point_cloud" /> -->
    

    <!-- Odom Node -->
    <!-- Launch odom node first to wait for data from registration node -->
    <node pkg="slam" type="laserOdometry_node" name="laserOdometry_node" output="screen" />

    <!-- Registration Node -->
    <node pkg="slam" type="scanRegistration_node" name="scanRegistration_node" output="screen" />

    <!-- Mapping node -->
    <node pkg="slam" type="laserMapping_node" name="laserMapping_node" output="screen" />

</launch>
